external/os-autoinst-common/.editorconfig

```diff
@@ -0,0 +1,12 @@
+[*]
```

Comment:
I just stumbled upon this issue: https://github.com/mvdan/sh/issues/664
And it mentions `[[shell]]`. I can't find that in the [shfmt documentation](https://manpages.debian.org/testing/shfmt/shfmt.1.en.html) nor in the [EditorConfig documentation](https://editorconfig.org/) but it seems to work :)

The problem seems to be that eslint is using the editorconfig as well, and as we are using `*` we can't blame it :)

Reply:
Oh, nevermind, I misread the diff.
It was apparently touching the right files when i used [[shell]], but then it was using the default formatting rules, using tabs etc.

Reply:
I tried `[script/**]` instead, but it still touched files elsewhere (and did not touch anything under script). Weird. Have to work on other tasks now though...

Reply:
Apparently this works, but we can't use the config file then:
```
shfmt -w -i 4 -bn -ci -sr $(shfmt -f .)
```
e.g.
```
shfmt -w $(shfmt -f .)
```
does not use .editorconfig

So -f is able to find the right files.

Reply:
What about the other way around?
```
[*.js]
ignore = true
```